crates/cheatnet/tests/common/state.rs

```diff
@@ -0,0 +1,25 @@
+use camino::Utf8PathBuf;
+use cheatnet::CheatnetState;
+use include_dir::{include_dir, Dir};
+use std::path::PathBuf;
+use tempfile::{tempdir, TempDir};
+
+static PREDEPLOYED_CONTRACTS: Dir = include_dir!("crates/cheatnet/predeployed-contracts");
+
+fn load_predeployed_contracts() -> TempDir {
+    let tmp_dir = tempdir().expect("Failed to create a temporary directory");
+    PREDEPLOYED_CONTRACTS
+        .extract(&tmp_dir)
+        .expect("Failed to copy predeployed contracts to temporary directory");
+    tmp_dir
+}
+
+#[allow(clippy::module_name_repetitions)]
+pub fn create_cheatnet_state() -> CheatnetState {
```

Comment:
Isn't that how we always want to instantiate `CheatnetState`? 
If that's the case, can we add the instantiation method to the struct?

Reply:
Depends whether we want to keep automatically add those predeployed contracts, perhaps it's okay, I'm not sure which version is better for forge.

Reply:
I'd keep it as is on this PR